UnityProject/Assets/Scripts/RobotScript.cs

```diff
@@ -23,6 +23,8 @@ public class RobotScript:MonoBehaviour{
     public AudioClip sound_engine_loop;
     public AudioClip sound_damaged_engine_loop;
 
+    public Transform point_pivot, top_rotor, bottom_rotor, point_spark, camera_pivot, battery, gun_pivot, point_muzzle_flash, gun_camera, drone_camera, motor;
```

Comment:
Don't think this shortcut has been used anywhere else in Receiver's code,

Reply:
I agree, probably looks better if each transform is declared on a separate line for consistency.